libbeat/cmd/instance/beat.go

```diff
@@ -118,6 +118,9 @@ type beatConfig struct {
 	Pipeline   pipeline.Config `config:",inline"`
 	Monitoring *common.Config  `config:"xpack.monitoring"`
 
+	// central managmenet settings
+	Management *common.Config `config:"xpack.management"`
```

Comment:
Should we really prefix the `management` setting by `xpack`. What about just making it `management`? I think long term it should also disappear from monitoring.

Reply:
This a good point, I'm comfortable with changing this to just `management`, I just followed the same path as `xpack.monitoring`

Reply:
I'll change it to `management`, please shout out if you are against it